codespell_lib/data/dictionary.txt

```diff
@@ -10707,8 +10742,10 @@ sortings->sorting
 sortlst->sortlist
 sortner->sorter
 sortnr->sorter
+sortrage->storage
 soruce->source
 soruces->sources
+sotrage->storage
```

Comment:
Can also be `shortage`

Reply:
Done, although this one feels a bit of a leap to me.